docs/guides/audit-seccomp.md

```diff
@@ -0,0 +1,110 @@
+---
+title: 'The "audit-seccomp" gadget'
+weight: 10
+---
+
+The Audit Seccomp gadget provides a stream of events with syscalls that had
+their seccomp filters generating an audit log. An audit log can be generated in
+one of those two conditions:
+
+* The Seccomp profile has the flag `SECCOMP_FILTER_FLAG_LOG` (currently
+  [unsupported by runc](https://github.com/opencontainers/runc/pull/3390)) and
+  returns any action other than `SECCOMP_RET_ALLOW`.
+* The Seccomp profile does not have the flag `SECCOMP_FILTER_FLAG_LOG` but
+  returns `SCMP_ACT_LOG` or `SCMP_ACT_KILL*`.
+
+### On Kubernetes
+
+* Install the Seccomp Operator.
+
+* Install a SeccompProfile that log the `mkdir` and `unshare` syscalls.
+
+```
+apiVersion: security-profiles-operator.x-k8s.io/v1beta1
+kind: SeccompProfile
+metadata:
+  name: log
+  annotations:
+    description: "Log some syscalls"
+spec:
+  defaultAction: SCMP_ACT_ALLOW
+  architectures:
+  - SCMP_ARCH_X86_64
+  syscalls:
+  - action: SCMP_ACT_KILL
+    names:
+    - unshare
+  - action: SCMP_ACT_LOG
+    names:
+    - mkdir
+```
+
+* Start a pod with that SeccompProfile.
+
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mypod
+spec:
+  securityContext:
+    seccompProfile:
+      type: Localhost
+      localhostProfile: operator/default/log.json
+  restartPolicy: Never
+  containers:
+  - name: container1
+    image: busybox
+    command: ["sh"]
+    args: ["-c", "sleep infinity"]
+```
+
+* Start the audit-seccomp gadget and execute the aforementioned syscalls in the
+  pod.
+
+```
+$ kubectl gadget audit-seccomp -o custom-columns=namespace,pod,syscall,code
```

Comment:
Maybe we should split this in two code blocks:
1. One showing there is no output because the gadget waits for syscalls to be executed.
2. Another which shows the syscalls after the below `exec` was run.

Reply:
Updated